source/bluetooth/MicroBitBLEManager.cpp

```diff
@@ -790,3 +841,24 @@ void MicroBitBLEManager::showNameHistogram(MicroBitDisplay &display)
             display.image.setPixelValue(MICROBIT_DFU_HISTOGRAM_WIDTH - i - 1, MICROBIT_DFU_HISTOGRAM_HEIGHT - j - 1, 255);
     }
 }
+
+/**
+ * Restarts in BLE Mode
+ *
+ */
+ void MicroBitBLEManager::restartInBLEMode(){
+   KeyValuePair* BLEMode = storage->get("BLEMode");
```

Comment:
Can we rename to "RebootMode" or similar, so this functionality is more general purpose?